src/lolex-src.js

```diff
@@ -611,32 +616,25 @@ function createClock(now, loopLimit) {
 }
 exports.createClock = createClock;
 
-exports.install = function install(target, now, toFake, loopLimit) {
-    var i, l;
-
-    if (target instanceof Date) {
-        toFake = now;
-        now = target.getTime();
-        target = null;
-    }
-
-    if (typeof target === "number") {
-        toFake = now;
-        now = target;
-        target = null;
-    }
-
-    if (!target) {
-        target = global;
-    }
+/**
+ * @param config {Object} optional config
+ * @param config.target {Object} the target to install timers in (default `window`)
+ * @param config.now {number|Date}  a number (in milliseconds) or a Date object (default epoch)
```

Comment:
All the current parameters are optional, `lolex.install()` is used in the README, so this behaviour-preserving and should not break anything.

We could add more explicit defaults of course, before sending them on to `createClock` and such (that have their own defaults), but I felt that would perhaps obfuscate my intent in this PR. Could add that in another commit, of course.